cpp/CMakeLists.txt

```diff
@@ -165,6 +165,9 @@ add_library(rmm SHARED
             src/rmm/memory_manager.cpp
             thirdparty/cnmem/src/cnmem.cpp)
 
+#Override RPATH for cudf & rmm
+SET_TARGET_PROPERTIES(cudf rmm PROPERTIES BUILD_RPATH "\$ORIGIN")
```

Comment:
I am not familiar with RPATH or $ORIGIN. Can you explain what this does?

Reply:
RPATH is the runtime search path for linked libraries. `$ORIGIN` means the current directory the library is in. For example, `libcudf.so` will search its current directory in addition to system directories (like `LD_LIBRARY_PATH` etc) for `librmm.so`.